cmd/admin-config-get.go

```diff
@@ -38,23 +40,21 @@ FLAGS:
   {{range .VisibleFlags}}{{.}}
   {{end}}
 EXAMPLES:
-  1. Get server configuration of a MinIO server/cluster.
-     {{.Prompt}} {{.HelpName}} play/
+  1. Get the current status of 'worm' mode on MinIO server/cluster.
+     $ {{.HelpName}} play/ worm
+     state="off"
```

Comment:
can we add some more examples on querying the subsystems?

Reply:
Definitely its not done yet @Praveenrajmani